ui/app/components/gas-customization/gas-modal-page-container/gas-modal-page-container.container.js

```diff
@@ -178,3 +197,25 @@ function addHexWEIsToRenderableFiat (aHexWEI, bHexWEI, convertedCurrency, conver
     partialRight(formatCurrency, [convertedCurrency]),
   )(aHexWEI, bHexWEI)
 }
+
+function getRenderableTimeEstimate (currentGasPrice, priceAndTimeEstimates) {
+  const gasPrices = priceAndTimeEstimates.map(({ gasprice }) => gasprice)
```

Comment:
Is the casing for the `gasprice` property correct? Is that coming from the API?

Reply:
Yes.